test/integration/requests/gatewayBalances.ts

```diff
@@ -0,0 +1,48 @@
+import { assert } from 'chai'
+import _ from 'lodash'
+
+import { GatewayBalancesRequest } from 'xrpl-local'
+
+import serverUrl from '../serverUrl'
+import { setupClient, suiteClientSetup, teardownClient } from '../setup'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('gateway_balances', function () {
+  this.timeout(TIMEOUT)
+
+  before(suiteClientSetup)
+  beforeEach(_.partial(setupClient, serverUrl))
+  afterEach(teardownClient)
+
+  it('base', async function () {
+    const request: GatewayBalancesRequest = {
+      command: 'gateway_balances',
+      account: 'rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh',
+      ledger_index: 'validated',
+      strict: true,
```

Comment:
It doesn't really matter for this, but the base request should technically be with no additional variables.